From 2b5885d7860250e16c7a314c9fdc5643484a43fc Mon Sep 17 00:00:00 2001
From: Ravi Vembu <raviv@motorola.com>
Date: Mon, 4 Jan 2016 22:01:24 -0600
Subject: [PATCH 588/959] IKSWM-15556: defconfig: mot-8952: Enable ALSA to H2W
 headset events converter

Enable ALSA to H2W headset events converter needed for Android
headset observer.

Change-Id: I4fcfc7d7ff3661179ead00c513bce9e054c65af4
Signed-off-by: Ravi Vembu <raviv@motorola.com>
Reviewed-on: http://gerrit.mot.com/810576
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 30cf992cc6d..b923e6f10b3 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -57,3 +57,4 @@ CONFIG_EPL8802=y
 CONFIG_SENSORS_FPC_1020=y
 CONFIG_MMC_FFU=y
 CONFIG_SUSPEND_DEBUG=y
+CONFIG_ALSA_TO_H2W=y
-- 
2.11.0

